.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "aion_fastvm"]
 	path = aion_fastvm
 	url = https://github.com/aionnetwork/aion_fastvm
+    branch = add_optional2db
```

Comment:
if the PR require using special vm branch, should consider merging the branch to the vm master.

Reply:
this merge should merge to dev, not master-pre-compile if this required by the precompiled-contract